packages/components/select/src/useSelect.ts

```diff
@@ -730,7 +730,9 @@ export const useSelect = (props, states: States, ctx) => {
     if (!Array.isArray(states.selected)) return
     const option = states.selected[states.selected.length - 1]
     if (!option) return
-
+    if (option.isDisabled) {
+      return option.isDisabled
+    }
```

Comment:
## Opinion

I think this fix introduces another problem where you can't delete a disabled item, but you can't access the old one for a disabled item by default.

<img width="241" alt="image" src="https://user-images.githubusercontent.com/27342882/217980137-99acfa91-8c78-4c81-a762-447beb13180a.png">

Unable to access with delete key (backspace)

Reply:
disabled option can't  clearable, it also can't delete by backspace key when enables filtering

Reply:
when I set the default value, it cannot be clearable, but it can be bankspace, which is unreasonable

Reply:
Yes, as you said, disabled options are not accessible. However, in the current PR situation, option 1 appears to be inaccessible (option 1 is not disabled).

I'm not sure how this works for the user experience.